src/docs/testing/ui-performance/index.md

```diff
@@ -412,49 +388,51 @@ At this point, disable the graphs and checkerboardOffScreenLayers.]
 
 Run the app and look for images rendered with a randomly colored checkerboard,
 indicating that the image is cached. As you interact with the scene,
-the checkerboarded images should remain constant—you don’t want to see
+the checkerboarded images should remain constant&mdash;you don’t want to see
 flickering, which would indicate that the cached image is being re-cached.
 
 In most cases, you want to see checkerboards on static images,
 but not on non-static images.  If a static image isn't cached,
-you can cache it by placing it into a
-[RepaintBoundary]({{site.api}}/flutter/widgets/RepaintBoundary-class.html)
+you can cache it by placing it into a [RepaintBoundary][]
 widget. Though the engine may still ignore a repaint boundary if it
 thinks the image isn't complex enough.
 
+### Viewing the widget rebuild profiler
+
+You can view the widget rebuilt counts for the current screen and
```

Comment:
Maybe provide a little more sell here of why this is useful.
E.g.
The Flutter framework is designed to make it hard to create applications that are not 60fps and smooth. Often if you have jank, it is because there is a simple bug causing more of the UI to be rebuilt each frame than you expected. The Widget rebuild profiler helps you debug and fix performance problems due to these sorts of bugs.